### ISSUE TITLE
CFLogin Issue On Windows

### ISSUE THREAD

**Issue by SHAHNUR00** [gabrielsimoes/cfparser.vim#23]
When I am trying to Login It shows me below message.
Error detected while processing function cfparser#CFLogin[11]..cfparser#CFGetToken: line    3: E684: list index out of range: 1 E15: Invalid expression: match[1] 

Need Help

**Comment by Ferriccc** [comment]
yes exactlly it works on linux but on windows it gives the same error if you find the solution please tell me.
